src/status_im/navigation/events.cljs

```diff
@@ -188,3 +188,25 @@
   (let [view-id (if (= view-id :shell-stack) (shell.utils/calculate-view-id) view-id)]
     {:db             (assoc db :view-id view-id)
      :set-view-id-fx view-id}))
+
+(defn navigate-wizard-next-screen
+  [db flow-config current-screen]
+  (first (filter (fn [screen]
+                   (let [skip-step (:skip-step? screen)]
+                     (and (not= (:screen-id screen) current-screen)
+                          (not (when (not (nil? skip-step)) (skip-step db))))))
+                 flow-config)))
+
+(rf/reg-event-fx
```

Comment:
imo it would be nice to have three separate events here.

:navigation/wizard-start
:navigation/wizard-next-step
:navigation/wizard-previous-step

as it's quite obvious from the event of what is happening then.

the lifecycle events of each step can be then easily passed as params if needs be:
e.g `on-start`, `on-next`, `on-previous`, there can also be an on-finish if we need that, but for now it seems not needed 🤔